src/components/Error.astro

```diff
@@ -0,0 +1,96 @@
+<script src="https://harshkhandeparkar.github.io/typoist/dist/typoist.min.js"></script>
```

Comment:
If the script is being used can remove the typoist package?

Reply:
lmao. @SachdevJai use the package not this.